docs/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@aws-amplify/ui-react-v1": "npm:@aws-amplify/ui-react",
-    "@codesandbox/sandpack-react": "^0.1.15",
+    "@codesandbox/sandpack-react": "0.1.9",
```

Comment:
Just curious, was there something broken in 0.1.15? This is where I wish it was possible to put comments in JSON.

Reply:
Yes, it just flat out wouldn't work. See https://github.com/codesandbox/sandpack/issues/135

Reply:
Looks like it was fixed in an update, will update the version

Reply:
Never mind, I tried it out and it didn't work :(

Reply:
**EDIT** Just saw that you've re-introduced Sandpack. Thank you!!!

---

@dbanksdesign We can remove this entirely – it was only used for the prototype & would need to be reimplemented once we adopt it again.

I see references to it in:

- `docs/package.json`
- `docs/src/pages/index.page.tsx`
- `docs/src/styles/index.scss`